notebooks/303-pytorch-int8-semseg-with-nncf-pot-openvino/303-pytorch-int8-semseg-with-nncf-pot-openvino.ipynb

```diff
@@ -40,13 +33,21 @@
     "!git clone https://github.com/alexgkendall/SegNet-Tutorial data"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "5def72ca",
+   "metadata": {},
+   "source": [
+    "The dataset is stored in CamVid subdirectory. It contains 701 .png images from dash cam."
+   ]
+  },
   {
    "cell_type": "markdown",
    "id": "24d32df7",
    "metadata": {},
    "source": [
-    "### Obtaining the uncompressed (FP32) performance and accuracy baselines\n",
-    "Download the FP32 pre-trained weights checkpoint for UNet on CamVid:"
+    "### Obtain the uncompressed (FP32) performance and accuracy baselines\n",
+    "Download the FP32 pre-trained model with weights checkpoint for UNet on CamVid:"
```

Comment:
This wording is not quite correct. The model is not being downloaded, just the weights for it - such is the way with PyTorch.